src/schemas/pricelist-json/pricesDataObject.ts

```diff
@@ -1,16 +1,11 @@
 import jsonschema from 'jsonschema';
 
-export const pricelistSchema: jsonschema.Schema = {
-    $schema: 'http://json-schema.org/draft-07/schema#',
-    $ref: '#/definitions/PricesDataObject',
-    definitions: {
-        PricesDataObject: {
-            type: 'object',
-            additionalProperties: {
-                $ref: '#/definitions/EntryData'
-            }
-        },
-        EntryData: {
+export const pricesDataObject: jsonschema.Schema = {
+    id: 'prices-data-object',
+    type: 'object',
+    patternProperties: {
+        // eslint-disable-next-line prettier/prettier, no-useless-escape
+        '^(\d+);([0-9]|[1][0-5])(;((uncraftable)|(untradable)|(australium)|(festive)|(strange)|((u|pk|td-|c|od-|oq-|p)\d+)|(w[1-5])|(kt-[1-3])|(n((100)|[1-9]\d?))))*?$': {
```

Comment:
I don't thing, that regex is needed here.

Reply:
so just `^(.)*$` ?

Reply:
I'm not that sure how does the validator work, but if there is just an option to use `string` that would be best

Reply:
not work with the new structure